package.json

```diff
@@ -1,36 +1,44 @@
 {
   "name": "fxa-auth-db-mem",
-  "version": "0.33.0",
   "description": "Memory backend (for testing) for the fxa-auth-db-server",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mozilla/fxa-auth-db-mem"
-  },
-  "scripts": {
-    "test": "grunt && ./scripts/tap-coverage.js test/backend test/local",
-    "start": "node bin/server.js",
-    "outdated": "npm outdated --depth 0"
-  },
+  "version": "0.33.0",
   "author": "Mozilla (https://mozilla.org/)",
-  "homepage": "https://github.com/mozilla/fxa-auth-db-mem",
   "bugs": {
     "url": "https://github.com/mozilla/fxa-auth-db-mem/issues"
   },
-  "license": "MPL-2.0",
   "dependencies": {
     "bluebird": "2.2.2",
-    "convict": "0.6.1",
+    "convict": "0.8.0",
```

Comment:
@pdehaan updating this version is too dangerous here, this is not related to this PR either.  Should probably not be done here